node/network/statement-distribution/src/lib.rs

```diff
@@ -1128,7 +1108,7 @@ async fn retrieve_statement_from_message<'a>(
 								);
 
 								if let Some(statement) = statement {
-									return Some((active_head, statement))
+									return Some(statement)
```

Comment:
Why do we return the `Seconded` statement as opposed to returning whatever is stored in `message`. It seems that `message` gets dropped in this branch?

Reply:
nvm, I misread the code and this whole thing is a match on `message`